external/VMS_Custom_Setup_Scripts/setup_scripts/vms_setup.sh

```diff
@@ -0,0 +1,33 @@
+##
+  # Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This indentation isn't the usual practice IMU

Reply:
Resolved this concern in the updated PR.